### PR TITLE
Added gen 8 pokedex data

### PR DESCRIPTION
**Pull request by hostedposted**
I added all sprites, hires, thumbnails and updated most of the json. Some values might have the null value if I did not finish scraping them yet.

### CODE REVIEW

**Comment by Purukitto** [comment]
Is this still a WIP or have you finished scarping

**Comment by hostedposted** [comment]
Still a WIP. I just have to scrape one more thing then it will be finished.

**Comment by hostedposted** [comment]
Just finished scraping the data.

**Comment by Purukitto** [comment]
Alright I'll have a look asap

File: pokedex.json [hunk 1]
```diff
@@ -45006,7 +45006,7 @@
     "description": "It evolved after experiencing numerous fights. While crossing its arms, it lets out a shout that would make any opponent flinch.",
     "evolution": {
       "prev": [
-        "linoone",
+        "264",
         null
```
Comment (Purukitto) [inline pokedex.json]: There are still a few nulls left in the json for example here

Comment (hostedposted) [reply pokedex.json]: I just removed all of the nulls and replaced them with the values they should be.

**Comment by hostedposted** [comment]
I removed all the nulls.

**Comment by Purukitto** [comment]
Looks good and done, I will cross verify with the API repo to ensure nothing is breaking and then merge this one

**Review by Purukitto — APPROVED** [review]

**Comment by Purukitto** [comment]
@hostedposted I found a minor issue in the new additions (i.e. All entries after 809) have the thumbnail url wrong.

The URL is mentioned as https://raw.githubusercontent.com/Purukitto/pokemon-data.json/master/images/pokedex/thumbnail/820.png where as the correct url will be https://raw.githubusercontent.com/Purukitto/pokemon-data.json/master/images/pokedex/thumbnails/820.png

In the url the `thumbnail` should be replaced with `thumbnails`

Let me know if you would like to start a new PR to fix this minor issue! Thanks
Refer #16

**Pull request merged**